src/core/pretty.ml

```diff
@@ -234,13 +243,11 @@ let pp_command : p_command pp = fun oc cmd ->
       List.iter (out " %a" pp_p_arg) args;
       Option.iter (out " : @[<hov>%a@]" pp_p_term) ao;
       out " ≔ @[<hov>%a@]@]" pp_p_term t
-  | P_inductive(ms,s,t,tl)   ->
-      out "@[<hov 2>%ainductive %a"
-        (Format.pp_print_list pp_modifier) ms pp_ident s;
-      out " :@ @[<hov>%a@] ≔@ \n  " pp_p_term t;
-      let pp_cons oc (id,a) =
-        Format.fprintf oc "%a:@ @[<hov>%a@]" pp_ident id pp_p_term a in
-      List.pp pp_cons "\n| " oc tl
+  | P_inductive(_, [])             -> ()
+  | P_inductive(ms,i::il)          ->
+      out "%a" (Format.pp_print_list pp_modifier) ms;
```

Comment:
This case analysis should be done in pp_p_inductive.

Reply:
Idem for P_rules?

Reply:
ok

Reply:
Done